### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Sensory Characteristics] [MAS1.3.3] [Tooltip is not provided for social media icons and Anchor Icon.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#686]
**User Experience:**
User who rely on AT will not get proper information about “social media icons and Anchor Icon ” if tool tip is not provided for those sensory characteristics.
 
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
**Attachments:**
[MAS1.3.3_Tooltip.zip](https://github.com/OfficeDev/script-lab/files/1445207/MAS1.3.3_Tooltip.zip)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to “social media icons and Anchor Icon“.
3. Verify tooltip is provided for “social media icons and Anchor Icon“ or not. 
 
**Current Behavior:** 
Tooltip is not provided for social media icons and Anchor Icon.

**Expected Behavior:**
Tooltip should be  provided for social media icons and Anchor Icon so that user who can see those sensory characteristics will get to know the information about those controls.

**Note:**
1. Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Tooltip is not provided for social media icons and anchor icon.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1823da5a-e8fb-4e63-b881-7eccb312c358}
**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.3
